src/NuGet/Microsoft.Orleans.OrleansTelemetryConsumers.Counters.nuspec

```diff
@@ -1,33 +1,31 @@
-﻿<?xml version="1.0"?>
+<?xml version="1.0"?>
 <package xmlns="http://schemas.microsoft.com/packaging/2010/07/nuspec.xsd">
   <metadata>
-    <id>Microsoft.Orleans.CounterControl</id>
```

Comment:
@jason-bragg OK, I'll not arg anymore... I want to finish this work... But just for the sake of information, there is no difference at all from running a console app and using installutil.exe specially the way I did, which uses exactly same code as the console app and installutil.exe is just calling the `Install()` method the same way the console does. But ok, will not enter in the debate. I just want this .Net Core blocker to go away. I'm going to revert everything and reapply some of the changes.